### DISCUSSION TITLE
Is there a way to convert queries to graphql ?

### DISCUSSION THREAD

**Discussion by farridav** [react-querybuilder/react-querybuilder#790] [Q&A]
I'm loving this project, great work guys !

for my use case, i was hoping to use this project to generate a query filter to use on a graphql endpoint (specifically shopify) specs here: https://shopify.dev/docs/api/usage/search-syntax

Do you think that any of the existing formats would fit in here? or is it possible to make a new one ?

Thanks again for the great work!

**Comment by jakeboone02** [comment]
This https://github.com/react-querybuilder/react-querybuilder/discussions/532#discussioncomment-6199522 is from a question about the `jq` format, but is relevant:

> I haven't used jq but after looking quickly at the syntax, I have the same concerns about it as I do with GraphQL, namely that the "what" can be intermingled with the "which". RQB really only deals with the "which" (like the `WHERE` clause in SQL), not the "what" (like the `SELECT` clause in SQL).

If there is a way to produce a purely condition-centric output, without including the "what"/`SELECT` part, I would love to be able to support GraphQL.

**Reply by jakeboone02** [reply]
Oh and thank you for the kind words :)
